app/src/main/java/de/westnordost/streetcomplete/data/elementfilter/ElementFilterExpression.kt

```diff
@@ -19,9 +33,9 @@ class ElementFilterExpression(
 
     /** returns whether the given element is found through (=matches) this expression */
     fun matches(element: Element): Boolean =
-        includesElementType(element.type)
-        && (element.tags.isNotEmpty() || mayEvaluateToTrueWithNoTags)
-        && (elementExprRoot?.matches(element) ?: true)
+        includesElementType(element.type) &&
+            (element.tags.isNotEmpty() || mayEvaluateToTrueWithNoTags) &&
+            (elementExprRoot?.matches(element) ?: true)
```

Comment:
Why would that be better than previously?

Reply:
It isn't; we already came to that conclusion in the summary. I didn't want to start with reverting specific commits just yet though.